my-first-radanalytics-app.adoc

```diff
@@ -1,10 +1,10 @@
-= My First RADanalytics Application
+= SparkPi, your first Apache Spark application with radanalytics.io
```

Comment:
"SparkPi: an introduction to applications using Apache Spark and source-to-image with radanalytics.io"

Is 'radanalytics.io' meant to encompass "oshinko and our tooling" in this blurb?

Reply:
so, i tested out this title but it seems really long on the page when it renders out. how would you feel about "SparkPi: an introduction to application using Apache Spark with radanalytics.io" ?

Reply:
that sounds good